### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add the refactored code in the development branch.

Code refactoring mainly focused on
- removing unnecessary class and code
- optimizing image loading using loading=lazy attribute, and alt attribute

There is no visual change that this PR will do.

### And It would be very nice if you can comment on the UI/UX of [marvelsQuiz](https://deploy-preview-5--marvelsquiz.netlify.app/)

### CODE REVIEW

**Review by Anurag021 — APPROVED** [review]
code looks good to me, UI looking great & good use of memes 😉, 
found too many use of class= "kushan-font-family", but I understand it's making UI look better.

File: pages/avengers-infinity-war/question-five.html [hunk 1]
```diff
@@ -64,16 +61,16 @@
           <div class="align-i-ctr flex justify-c-sb mb-2 px-0p5">
             <span class="fs-2">
               <span class="fw-bold">
-                <span class="kushan-font-family text-primary">Q</span>
```
Comment (MayankKumar10) [inline pages/avengers-infinity-war/question-five.html:67]: add your font family in HTML tag so it will inherit by its child components

**Review by MayankKumar10 — APPROVED** [review]
Else everything looks good to me.

File: pages/avengers-infinity-war/question-five.html (hunk 1 above)
Comment (hsnice16) [reply pages/avengers-infinity-war/question-five.html:67]: Thanks for checking this. Actually, the base font family is Lato which is coming from the CL. This font family, I need for only some characters. That's why I am doing it like this.

Comment (MayankKumar10) [reply pages/avengers-infinity-war/question-five.html:67]: I guess you use this more so make it base family

Comment (hsnice16) [reply pages/avengers-infinity-war/question-five.html:67]: Actually, I have used it only for a single character, wherever I have used it.

File: index.html [hunk 2]
```diff
@@ -88,9 +79,10 @@ <h1 class="fs-3 text-center">Choose Your Category</h1>
...
             <div
               id="category-avengers-infinity-war"
-              class="card card-text-overlay card-vertical overflow-hidden shadow-light w-33"
+              class="card card-text-overlay card-vertical shadow-light"
             >
               <img
+                loading="lazy"
                 class="card-vertical-img"
                 src="assets/pngs/avenger-infinity.png"
                 alt="avengers infinity war poster png"
```
Comment (anik31) [inline index.html]: Just a suggestion to not include poster png in the image alt, you can just include text that's part of image.

**Review by anik31 — APPROVED** [review]
UI is nice & the code structure, naming everything is good.

File: index.html (hunk 2 above)
Comment (hsnice16) [reply index.html]: Thanks, @anik31, for pointing this out. I will look into it.

**Comment by hsnice16** [comment]
Thank you, for reviewing this.

**Pull request merged**